support/mock/mockrt.go

```diff
@@ -400,6 +400,11 @@ func (rt *Runtime) SetBalance(amt abi.TokenAmount) {
 	rt.balance = amt
 }
 
+// Get balance outside of a runtime call for test purposes
+func (rt *Runtime) GetBalance() abi.TokenAmount {
+	return rt.balance
+}
```

Comment:
I don't think you need this: just call CurrentBalance()

Reply:
This was to avoid using a constant or scattering bare ints around, to know the balance we created down in requireCreateWithLanes. `CurrentBalance` fails in such a context, b/c it calls `requireInCall`. I could remove that requirement but all of the mocked API calls have it, so I hesitated to change it.  I could simply return the balance used I suppose.... I'll do that instead.

Reply:
Ah, ok. GetBalance is fine.